xbmc/cores/VideoPlayer/VideoRenderers/WinRenderer.h

```diff
@@ -235,7 +235,7 @@ class CWinRenderer : public CBaseRenderer
   unsigned int         m_destHeight;
 
   int                  m_neededBuffers;
-  unsigned int         m_frameIdx;
+  unsigned int         m_frameIdx = 0;
```

Comment:
The code contains comparisons that don't take wraparound into account, so 0 seems much safer